src/main/java/org/broadinstitute/hellbender/Main.java

```diff
@@ -57,20 +61,8 @@
     private static final int ANY_OTHER_EXCEPTION_EXIT_VALUE = 3;
     private static final String STACK_TRACE_ON_USER_EXCEPTION_PROPERTY = "GATK_STACKTRACE_ON_USER_EXCEPTION";
 
-    /**
-     * The packages we wish to include in our command line.
-     */
-    protected List<String> getPackageList() {
-        final List<String> packageList = new ArrayList<>();
-        packageList.addAll(Arrays.asList("org.broadinstitute.hellbender"));
-        return packageList;
-    }
-
-    /**
-     * The single classes we wish to include in our command line.
-     */
-    protected List<Class<? extends CommandLineProgram>> getClassList() {
-        return Collections.emptyList();
+    protected GATKConf getConfiguration() {
+        return GATKConfBuilder.getDefaultConfiguration();
```

Comment:
I think it would be better if the master configuration object lived as a field in `CommandLineProgram` accessible by subclasses via an inherited method. This would simplify passing property-based information around the engine in the future.